src/nav/MainNavBar.js

```diff
@@ -49,7 +49,7 @@ class MainNavBar extends PureComponent {
     const leftPress = editMessage ? actions.cancelEditMessage : onPressStreams;
 
     return (
-      <View style={[styles.navBar, { backgroundColor }]}>
+      <View style={[styles.navBar, { backgroundColor, zIndex: 1 }]}>
```

Comment:
And here too.
We have to pass an object of `{ backgroundColor }` as it is a dynamic value.